torch/csrc/jit/passes/batch_mm.cpp

```diff
@@ -260,8 +260,134 @@ void BatchMMBlock(Block* block) {
   }
 }
 
+bool shape_is_fast_for_side(const at::Tensor& other_side_input) {
+  // Cutoff chosed by benchmarking on a TITAN V
```

Comment:
What is considered to be "fast"? Is it if something turns out to be faster when doing BatchMM vs not doing BatchMM?

Reply:
Yep. I basically those a cutoff at the boundary where the batched version is faster.